src/main/scala/ing/wbaa/druid/client/DruidAdvancedHttpClient.scala

```diff
@@ -283,7 +284,10 @@ object DruidAdvancedHttpClient extends DruidClientBuilder {
     val retryDelay =
       FiniteDuration(clientConfig.getDuration(Parameters.QueryRetryDelay).getNano, NANOSECONDS)
 
-    val brokerFlows = createConnectionFlows(druidConfig.hosts, druidConfig.secure, poolConfig)
+    val brokerFlows = createConnectionFlows(druidConfig.hosts,
+                                            druidConfig.secure,
+                                            druidConfig.responseParsingTimeout,
```

Comment:
Interesting, the `responseParsingTimeout` was already in the config, but we weren't using it 🤔